apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -19,10 +19,11 @@ export const studentSectionDataPropType = PropTypes.shape({
   userType: PropTypes.string,
   age: PropTypes.number,
   gender: PropTypes.string,
-  secretWords: PropTypes.string,
-  secretPictureName: PropTypes.string,
-  secretPicturePath: PropTypes.string,
+  secret_words: PropTypes.string,
+  secret_picture_name: PropTypes.string,
+  secret_picture_path: PropTypes.string,
```

Comment:
whats the reason for changing these to snake_case? Why are some of these fields snake_case and some camelCase (i.e. sectionId/loginType)?

Reply:
That's what they're named in the API - I could also map the names before I pass them in.

Reply:
That's what I might do. I always find it confusing to have a mixture of different naming conventions in a single object.